contract/area_types.go

```diff
@@ -11,3 +11,14 @@ type GetAreaTypesRequest struct {
 type GetAreaTypesResponse struct {
 	AreaTypes []model.AreaType `json:"area-types"`
 }
+
+// GetAreaTypeParentsRequest defines the schema for the GET /area-types/{area-type}/parents query parameter
+type GetAreaTypeParentsRequest struct {
+	Dataset  string `schema:"dataset"`
+	AreaType string `schema:"area-type"`
+}
```

Comment:
Why do we need this?
The enpoint is `GET /area-types/{area-type}/parents` and there are no query parameters.

Please remove this.